susemanager/susemanager.spec

```diff
@@ -217,7 +217,7 @@ cp -r pub/empty-deb %{buildroot}%{reporoot}/repositories/
 mkdir -p %{buildroot}%{_datadir}/YaST2/clients
 mkdir -p %{buildroot}%{_datadir}/YaST2/scrconf
 mkdir -p %{buildroot}%{_datadir}/applications/YaST2
-mkdir -p %{buildroot}/etc/YaST2
+mkdir -p %{buildroot}%{_sysconfdir}/YaST2
```

Comment:
Probably not needed, as we ditched the YaST2 wizards with the move to containers. But again, not for today, as everything still works.